docs/integrations/ios-specific-workflow-steps/convert-xcresult-to-xml-html.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Convert Xcresult to HTML/XML 
+metaTitle: Convert Xcresult to HTML/XML
+metaDescription: Convert Xcresult to HTML/XML
+sidebar_position: 1
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Convert Xcresult to HTML/XML
+
+After the [**Xcodebuild for Unit and UI Tests**](https://docs.appcircle.io/continuous-testing/running-ios-unit-and-ui-tests) step runs, it creates a test_result.xcresult file. In some specific cases, this test file needs to be converted to another format accordingly. The **Convert Xcresult to HTML/XML** step is used to convert this test file to `HTML` and `XML` formats.
+
+
+### Prerequisites
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [**Xcodebuild for Unit and UI Tests**](https://docs.appcircle.io/continuous-testing/running-ios-unit-and-ui-tests) | This step allows you to run unit and UI tests in your project. |
+
+:::caution
+Please note that if you do not run Xcodebuild for Unit and UI Tests before this step, the step will give an error because there will be no test result file to convert.
```

Comment:
I think easy to read when step name be bold.
```suggestion
Please note that if you do not run **Xcodebuild for Unit and UI Tests** before this step, the step will produce an error because there will be no test result file to convert.
```